src/app/static/src/js/components/reportLog/reportLog.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+    <div>
+        <h2>Running report logs</h2>
+        <div v-if="showReports" id="logs-form-group" class="form-group row">
+            <label class="col-sm-2 col-form-label text-right">Show logs for</label>
+            <div class="col-sm-6">
+                <running-reports-list id="logs" 
+                :reports="reports" 
+                v-on="$listeners"
+                :key.sync="selectedRunningReportKey"/>
```

Comment:
```suggestion
                :initial-selected-key="selectedRunningReportKey"/>
```
`<running-reports-list :key.sync="selectedRunningReportKey">` looks wrong because `runningReportsList` doesn't have a `key` prop. I guess the intention here is to ensure that correct report is selected in the dropdown, if it's passed in from the tabs component. The `sync` is unnecessary because the existing `v-on` will pass the `update:key` event up to `<run-report-tabs>`, which will pass it back down here via a prop (after a correction is made there - see other comment).

**However** I'm concerned that attempting to pre-select the correct in the dropdown via a new `initial-key` prop won't actually work because our existing dropdown widget only allows selection by text (i.e. what's actually displayed) and not by underlying data (i.e. the key). Even if that's the case I suggest adding the prop regardless and we can open a new ticket to investigate how to use it to pre-select the right value.

Reply:
Agree it's best to do this in a separate ticket. It's a nice to have, but I'm not sure it's completely necessary to actually pre-select the report in the drop-down when it comes in from the parent anyway - the details view will show the report name so there's no displayed information being lost by not showing it. The only downside is that it's inconsistent with what the user sees when they select from the drop-down themselves, and that if they want to go on to view a running version of the same report, the drop down won't be filtered to the report when they click on it.